### ISSUE TITLE
Input fields get confused

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Perundurai-Products/microemu#82]
```
What steps will reproduce the problem?
1. Open 
http://download.getjar.com/downloads/wap/export-88-202x533324819-20216859489/733
83/mig33v4.jad
or 
http://www.mig33.com/wap2/v4_30/24x24/mig33v4.jad
2. Try to login
3. The password field can't be entered properly, always stays empty and has the 
value of the first input field.

What version of the product are you using? On what operating system?
2.0.4

Please provide any additional information below.

```

Original issue reported on code.google.com by `jdd...@gmail.com` on 4 Nov 2010 at 8:55

**Comment by GoogleCodeExporter** [comment]
```
Please verify whether the problem exists in version 3.0.0 snapshot:
http://snapshot.microemu.org/microemulator/download/
```

Original comment by `bar...@gmail.com` on 4 Nov 2010 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
Hi, I have tested it with http://www.microemu.org/microemu-webstart/demo.jnlp 
(is it ok to use the webstart?)
And the app doesn't get past the Loading screen in this case.

If I use the snapshot .zip file:
I get the following when trying to launch the midlet:


java.lang.NoClassDefFoundError: javax/microedition/media/Controllable
    at com.projectgoth.Roshan.Main.<init>(Unknown Source)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
    at java.lang.reflect.Constructor.newInstance(Unknown Source)
    at java.lang.Class.newInstance0(Unknown Source)
    at java.lang.Class.newInstance(Unknown Source)
    at org.microemu.app.Common.loadMidlet(Common.java:418)
    at org.microemu.app.Common.initMIDlet(Common.java:1051)
    at org.microemu.app.launcher.Launcher.commandAction(Launcher.java:121)
    at javax.microedition.lcdui.Display$DisplayAccessor$1.run(Display.java:209)
    at org.microemu.device.ui.EventDispatcher$RunnableEvent.run(EventDispatcher.java:246)
    at org.microemu.device.ui.EventDispatcher.post(EventDispatcher.java:164)
    at org.microemu.device.ui.EventDispatcher.run(EventDispatcher.java:99)
    at java.lang.Thread.run(Unknown Source)
Caused by: 
org.microemu.app.classloader.MIDletClassLoader$LoadClassByParentException: 
javax.microedition.media.Controllable
    at org.microemu.app.classloader.MIDletClassLoader.findClass(MIDletClassLoader.java:342)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at org.microemu.app.classloader.MIDletClassLoader.loadClass(MIDletClassLoader.java:213)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    ... 15 more

```

Original comment by `jdd...@gmail.com` on 4 Nov 2010 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
It is necessary to include the microemu-jsr-135.jar from lib folder to the 
classpath. Unfortunately http://www.microemu.org/microemu-webstart/demo.jnlp 
doesn't include it.
```

Original comment by `bar...@gmail.com` on 4 Nov 2010 at 1:22

**Comment by GoogleCodeExporter** [comment]
```
Thanks for that, how about adding a simple .bat file containing?

java -jar microemulator.jar -cp lib\microemu-jsr-135.jar

a .sh equivalent if necessary.

This bug can be closed as FIXED in 3.0.0.

Regards,


David
p.s. do you get paid to work on this project?
```

Original comment by `jdd...@gmail.com` on 4 Nov 2010 at 1:28

**Comment by GoogleCodeExporter** [comment]
```
In final 3.0.0 I'm planning to include microemu-jsr-135 into microemulator.jar. 
Then only executing the microemulator.jar will be necessary.

Sometimes I get paid. It depends on who and how important is new feature or 
bugfix for the user.
```

Original comment by `bar...@gmail.com` on 4 Nov 2010 at 1:44
- Changed state: **Fixed**
- Added labels: **Milestone-3.0**
